routes/search.js

```diff
@@ -39,17 +42,20 @@ router.get('/', common.isAuthenticated, (req, res, next) => {
 // Get all avaialable books
 // ******** need to update this based on worldwide shippers
 function allBooks(id, callback){
-    var selectAllAvailableBooks = `SELECT UserBooks.id AS id, title, author, imgUrl, isbn13 AS isbn, genre, rating, pubDate, Users.country
-                                 FROM UserBooks
-                                 INNER JOIN Books ON Books.id = UserBooks.bookId
-                                 INNER JOIN Users ON Users.id = UserBooks.userId
-                                 WHERE UserBooks.userId != ? AND UserBooks.available = 1 AND Users.worldwide = 1
-                                 UNION
-                                 SELECT UserBooks.id AS id, title, author, imgUrl, isbn13 AS isbn, genre, rating, pubDate, Users.country
-                                 FROM UserBooks
-                                 INNER JOIN Books ON Books.id = UserBooks.bookId
-                                 INNER JOIN Users ON Users.id = UserBooks.userId
-                                 WHERE UserBooks.userId != ? AND UserBooks.available = 1 AND Users.country = ?`;
+    var selectAllAvailableBooks = `SELECT UserBooks.id AS id, listingDate, description, title AS title, author AS author, imgUrl AS imgUrl, isbn13 AS isbn, Users.country
```

Comment:
Don't need the AS here for the fields that don't change name, this is another refactor change that you are undoing with your work. I'd also add an AS for description (which I'm guessing is condition?) to make it more obvious that's what this is